index.html

```diff
@@ -201,7 +201,7 @@ <h5 class="intro2">LET'S CONNECT</h5>
         <a href="#"><img src="images/fingers.png" alt="fingers"></a>
     </li>
  </ul>
-
+ 
```

Comment:
- [ ] please kindly follow this design for your about me section of the desktop version
yours
![Screenshot (667)](https://user-images.githubusercontent.com/65298048/206855032-6f149b37-04e8-4261-a5e3-5cf47231d53b.png)
figma
![Screenshot (666)](https://user-images.githubusercontent.com/65298048/206855036-66ea117b-2a08-4543-bdf9-6df6a588815c.png)